depthai.py

```diff
@@ -13,12 +13,28 @@
 import subprocess
 import platform
 from pathlib import Path
+import struct
 
 import depthai
 
 import consts.resource_paths
 from depthai_helpers import utils
 
+
+class Tracklet:
+    states = {
```

Comment:
Each of these are [class variables](https://docs.python.org/3/tutorial/classes.html#class-and-instance-variables), but you're instantiating the class below and changing the class vars for each instance. Using class variables as instance variables can lead to some nasty bugs (see improper use of class variables in the link). Other suggestions on this:

Use a dataclass -- fast, modern, easy to read. Requires 3.7 though. If you want to support older versions of python, just use instance variables.
Use an enum for the states and remove from the class.

Reply:
I agree, partially. Dataclasses are great, but we've made some effort already to support python 3.5 (not sure if lower).

Moving the fields to `__init__` and making `states` even an inner class (so it's a class inside a class) would be enough I beleive, with Enums we'd have to add `.value` suffix to each entry

Reply:
Yeah, the suggestions were merely suggestions. I try to brainstorm solutions when reviewing, but please take anything with the word "suggestion" as just a minor improvement or maybe just an idea.

Reply:
Sure, thanks @rhollen! And sorry if I sounded too harsh or arogant, I'm not a native speaker, still learning. Just wanted to bring this up for the discussion :)